geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/DeploymentManagementController.java

```diff
@@ -63,7 +67,7 @@
       @RequestParam(required = false) String group) {
     Deployment deployment = new Deployment();
     if (StringUtils.isNotBlank(id)) {
-      deployment.setFileName(id);
+      deployment.setDeploymentName(id);
```

Comment:
I don't know much about this area...but thought of asking this; with these changes does an older rest client can work with newer cluster...

Reply:
Yes.. DeploymentName can be derived from the Filename if none is provided. This is to make sure that the gfsh command is backwards compatible as well.

Reply:
https://github.com/apache/geode/blob/16c7e34318101d619e438ce8b4707d19fba996e2/geode-management/src/main/java/org/apache/geode/management/configuration/Deployment.java#L122-L129

Reply:
@agingade good catch. We tried to make that backward compatibility is met.
I've added tests to support that now in https://github.com/apache/geode/pull/6144/commits/9d1af5af3d71b9491fcbf786bc8313c2137504e4#diff-824a5037676a6a6e678516d51503f74d2493e960080ada8b66911571836e722e